content/pages/entry.rst

```diff
@@ -9,14 +10,23 @@ Here are some of the topics that will be covered:
 
 Introductory Workshop
 ---------------------
-We'll go over the basics of Linux, the terminal, irc, and other awesome tools
-that we'll use throughout Devops Bootcamp.
-
-Single System Fundamentals Workshop
------------------------------------
-We'll explore the basics of Linux System fundamentals, including file
-permissions, users, groups, package management and a few other misc related
-topics.
+We'll go over what DevOpsBootcamp is all about and get you started with your
+own system that you can start following our lessons in. All that's required is
```

Comment:
Change ``that you can start following our lessons in`` to ``that you can use for hands on demos``, or something like that. Following our lessons in doesn't quite make sense.